src/Helper/ValidateStreet.php

```diff
@@ -69,6 +70,10 @@ public static function getStreetRegexByCountry(string $local, string $destinatio
             return self::SPLIT_STREET_REGEX_NL;
         }
 
+        if ($localIsNl && $destinationIsBe) {
```

Comment:
Als destination BE is, zal je nooit een belgisch adres willen opsplitsen op de nederlandse manier.

Reply:
Zou je me willen vertellen wat waarom je dit doet? Ik zie een story nummer in je pr titel.

Reply:
Ik denk dat bij NL -> BE het valide is dat getStreetRegexByCountry null terug geeft.